src/translate/options.py

```diff
@@ -49,6 +49,10 @@ def parse_args():
         "--keep-unimportant-variables",
         dest="filter_unimportant_vars", action="store_false",
         help="keep variables that do not influence the goal in the causal graph")
+    argparser.add_argument(
+        "--keep-noop-operators",
+        dest="filter_noop_operators", action="store_false",
+        help="keep noop operators in the output")
```

Comment:
Despite the previous comment on consistency, I suggest we do not follow the precedent set by `filter_unimportant_vars` in two ways. Firstly, I think it's clearer if the option sets a flag to true rather than to false. Secondly, "filter" is a poor word because it can either mean "keep" (as in Python's "filter" function) or "remove" (as in filter out, like here). I also prefer "no-op" over "noop" in user-facing text, and for consistency would then perhaps also use "no_op" for variable names.

Suggestion: rename option to "keep-no-ops" (no-op operators feels a bit redundant), with destination "keep_no_ops" using "store_true", flipping the meaning of the switch. Help text changed to "keep operators without effects in the output".